integration/maptest/map_test.go

```diff
@@ -211,6 +216,16 @@ func TestLeafHistory(t *testing.T) {
 	}
 }
 
+func mustMarshalMetadata(t *testing.T, in proto.Message) *any.Any {
```

Comment:
factor this out to `testonly/` so can reuse here and in `crypto/verifier_test.go`?